src/sentry/static/sentry/app/utils/ajaxCsrfSetup.tsx

```diff
@@ -1,13 +1,17 @@
 import getCookie from 'app/utils/getCookie';
 import {CSRF_COOKIE_NAME} from 'app/constants';
 
-function csrfSafeMethod(method) {
+function csrfSafeMethod(method?: string) {
   // these HTTP methods do not require CSRF protection
-  return /^(GET|HEAD|OPTIONS|TRACE)$/.test(method);
+  return /^(GET|HEAD|OPTIONS|TRACE)$/.test(method ?? '');
 }
 
-export default function ajaxCsrfSetup(xhr, settings) {
+export default function ajaxCsrfSetup(
+  this: JQueryAjaxSettings,
```

Comment:
FYI `this` is a shadow parameter https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#specifying-the-type-of-this-for-functions